roles/apache/tasks/install-RedHat.yml

```diff
@@ -1,7 +1,8 @@
 ---
 
 - name: Ensure Apache HTTP server is installed (CentOS)
-  yum: name={{ item }} state=present update_cache=yes
-  with_items:
-    - httpd
```

Comment:
Indeed it seems that iteration is redundant here. @NicolasLiampotis do you agree?